iroha/src/sumeragi.rs

```diff
@@ -605,8 +608,13 @@ impl Sumeragi {
         Ok(())
     }
 
-    async fn next_round(&mut self, prev_block_hash: Hash) {
-        self.network_topology.sort_peers(Some(prev_block_hash));
+    async fn commit_block(&mut self, block: SignedBlock) {
+        let block = block.validate(&*self.world_state_view.read().await);
+        self.latest_block_hash = block.hash();
+        self.block_height = block.header.height;
```

Comment:
Why just to do not store latest block?

Reply:
We do not always have a latest block. Besides the format of the latest block in Kura will be different from the latest block that we store in sumeragi: Kura has `CommittedBlock`, Sumeragi has only `ValidBlock`.